src/config/network/iwd.md

```diff
@@ -0,0 +1,74 @@
+# IWD
+
+[IWD](https://iwd.wiki.kernel.org/) (iNet Wireless Daemon) is a wireless daemon for Linux that aims to replace [WPA supplicant](./wpa_supplicant.md).
+
+## Installation
+
+Install the `iwd` package.
+
+```
+# xbps-install -S iwd
+```
+
+Enable the `dbus` and `iwd` services.
+
+```
+# ln -s /etc/sv/dbus /var/service
+# ln -s /etc/sv/iwd /var/service
+```
+
+By default, IWD can be controlled by the root user, or those in the `wheel` group.
+
+```
+# gpasswd -a <user> wheel
+```
+
+## Usage
+
+IWD provides a command line client, `iwctl`. It can be used to add, remove, and configure network connections; running it produces an interactive prompt. To list available options, enter `help`.
+
+```
+$ iwctl
+
+[iwd]# help
+                               Available commands
+--------------------------------------------------------------------------------
+  Commands                                          Description
+--------------------------------------------------------------------------------
+
+Adapters:
+  adapter list                                      List adapters
+  adapter <phy> show                                Show adapter info
+  adapter <phy> set-property <name> <value>         Set property
+...
+```
+
+Some useful commands:
+
+```
+[iwd]# device list                                       # List all Wi-Fi devices
+[iwd]# device <interface> show                           # Display details of a Wi-Fi device 
+[iwd]# station <interface> scan                          # Scan for networks
+[iwd]# station <interface> get-networks                  # List networks
+[iwd]# station <interface> connect <network-name>        # Connect to a network
+[iwd]# station <interface> connect-hidden <network-name> # Connect to a hidden network
+[iwd]# station <interface> disconnect                    # Disconnect from a network
+[iwd]# known-networks list                               # List known networks
+[iwd]# known-networks forget <network-name>              # Forget a known network
+```
+
+> Note: `iwctl` also supports passing commands from the interactive session as command line arguments.
```

Comment:
"supports passing commands as arguments" maybe? This is confusing because iwctl seems to function as an interactive session otherwise.

Reply:
Yeah, I really wasn't sure how to word this. Thanks